src/workflows/airqo_etl_utils/datautils.py

```
@@ -70,40 +74,17 @@ def extract_data_from_bigquery(
-                }
-            },
-        }.get(datatype, None)
-
         if not device_category:
             device_category = DeviceCategory.GENERAL
-
-        if source:
+        try:
+            source = configuration.DataSource.get(datatype)
             table = source.get(device_category).get(frequency)
+        except KeyError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)